Source/Blazorise/Components/MessageAlert/MessageAlert.razor.cs

```diff
@@ -86,7 +86,7 @@ protected Task OnCancelClicked()
         {
             return InvokeAsync( async () =>
```

Comment:
redudancy? Can extract inner delegate?

Reply:
This is OK. We have multiple operations that needs to be awaited.